### PR TITLE
feat: Add some small UI improvements

### PR DESCRIPTION
**Pull request by pedrofaria**
* Add sorting by number of players.
* Add session remaining time in the servers list.
image

* Move server name to the beggining of the create/update server settings.
image

### CODE REVIEW

**Comment by pedrofaria** [comment]
RC 1 
download link: [accweb_1.18.0-rc1.zip](https://github.com/assetto-corsa-web/accweb/files/8994412/accweb_1.18.0-rc1.zip)

File: public/src/pages/overview.vue [hunk 1]
```diff
@@ -50,6 +50,7 @@ export default {
                 "udp_port": {label: "UDP Port", field: "udpPort", order: "asc"},
                 "tcp_port": {label: "TCP Port", field: "tcpPort", order: "asc"},
                 "is_running": {label: "Is Running", field: "isRunning", order: "desc"},
+                "nr_clients": {label: "Drivers Quantity", field: "nrClients", order: "desc"}
```
Comment (matslindh) [inline public/src/pages/overview.vue:53]: "Driver count"?

Comment (pedrofaria) [reply public/src/pages/overview.vue:53]: I like your idea

File: public/src/components/server.vue [hunk 2]
```diff
@@ -25,7 +25,7 @@
                 <b>{{$t("state")}}: </b>{{$t(server.serverState)}} &bull;
                 <b>{{$t("number_of_drivers")}}: </b>{{formattedServerClientCount}} &bull;
                 <b>{{$t("session")}}: </b>
-                <span v-if="server.sessionType">{{server.sessionType}} ({{server.sessionPhase}})</span>
+                <span v-if="server.sessionType">{{server.sessionType}} ({{server.sessionPhase}}) - {{server.sessionRemaining}} min(s)</span>
```
[the remark this answers is not in the document]
Comment (pedrofaria) [reply public/src/components/server.vue:28]: don't think so

[the remark this answers is not in the document]
Comment (matslindh) [reply public/src/components/server.vue:28]:
```
{{server.sessionType}} ({{server.sessionPhase}}) - {{server.sessionRemaining}} min(s)</span>
                       ^                                                             ^  - nothing - ) here ends (s)

**Pull request merged**